code/game/objects/items/knives.dm

```diff
@@ -17,13 +18,15 @@
 	throw_speed = 3
 	throw_range = 6
 	custom_materials = list(/datum/material/iron = SHEET_MATERIAL_AMOUNT * 6)
-	attack_verb_continuous = list("slashes", "stabs", "slices", "tears", "lacerates", "rips", "dices", "cuts")
-	attack_verb_simple = list("slash", "stab", "slice", "tear", "lacerate", "rip", "dice", "cut")
+	attack_verb_continuous = list("slashes", "slices", "tears", "lacerates", "rips", "dices", "cuts")
+	attack_verb_simple = list("slash", "slice", "tear", "lacerate", "rip", "dice", "cut")
 	sharpness = SHARP_EDGED
 	armor_type = /datum/armor/item_knife
 	wound_bonus = 5
 	bare_wound_bonus = 15
 	tool_behaviour = TOOL_KNIFE
+	var/static/list/alt_continuous = list("stabs", "pierces", "shanks")
+	var/static/list/alt_simple = list("stab", "pierce", "shank")
```

Comment:
I'll note: Our existing attack verbs are cached via string_list, while these alternative ones you've set are just static - meaning we'll have some duplicate lists floating around

I think you should (probably) be passing them to the comp as `string_list(list("stabs", "pierces", "shanks"))` etc, but it's not the end of the world ultimately (since I think transforming just passes them raw without caching them at all (though maybe we should fix that))